### PR TITLE
Update train_loop.rst

### LINKED ISSUE #4696: Missing device allocation in "MNIST with a Manual Training Loop" tutorial
https://docs.chainer.org/en/stable/examples/train_loop.html

MNIST using Trainer on the other hand, has the device allocation. Without it, the manual training loop example runs into error.
> gpu_id = 0  # Set to -1 if you use CPU
model = MLP()
if gpu_id >= 0:
    model.to_gpu(gpu_id)

### PR DESCRIPTION
**Pull request by arisliang**
Fix #4696

### CODE REVIEW

File: docs/source/examples/train_loop.rst [hunk 1]
```diff
@@ -149,6 +149,11 @@ The main steps are twofold:
                 return self.l3(h)
 
         model = MyNetwork()
+        
+        gpu_id = 0 # Set to -1 if you use CPU
+        model = MLP()
```
[the remark this answers is not in the document]
Comment (arisliang) [reply docs/source/examples/train_loop.rst]: Yes, we should. Sorry I missed that. Line removed in the new commit.

**Comment by delta2323** [comment]
@Crissman I think the test environment does not have GPU. So, should we run the example code in CPU by default?

**Comment by arisliang** [comment]
changed default to cpu as requested.

**Comment by kmaehashi** [comment]
Test environment has a GPU, so you can use `gpu_id = 0` as in MNIST example.

File: docs/source/examples/train_loop.rst [hunk 2]
```diff
@@ -149,6 +149,10 @@ The main steps are twofold:
                 return self.l3(h)
 
         model = MyNetwork()
+        
+        gpu_id = -1 # Set to -1 if you use CPU, or >= 0 to use gpu.
```
Comment (kmaehashi) [inline docs/source/examples/train_loop.rst]: * Please use `gpu_id = 0`
* Please insert 2-spaces between code and comment (`#`)

**Comment by delta2323** [comment]
> Test environment has a GPU,

Sorry, I misunderstood.

**Comment by delta2323** [comment]
jenkins, test this please.

**Comment by delta2323** [comment]
jenkins, test this please.

**Comment by delta2323** [comment]
@kmaehashi  The CI passed. Could you double-check the PR and merge it if it is OK?

**Comment by kmaehashi** [comment]
LGTM!

**Pull request merged**